Gemfile

```diff
@@ -76,5 +76,7 @@ gem 'aws-sdk-rails'
 # Exceptions.
 gem 'exception_notification', git: 'https://github.com/smartinez87/exception_notification'
 
+#Search
+gem 'pg_search'
```

Comment:
a newline after this

Reply:
Added `.editorconfig` to take care of this.